src/coreclr/ilasm/asmparse.y

```diff
@@ -42,13 +42,13 @@
 };
 
         /* These are returned by the LEXER and have values */
-%token ERROR_ BAD_COMMENT_ BAD_LITERAL_                         /* bad strings,    */
+%token BAD_COMMENT_ BAD_LITERAL_                         /* bad strings,    */
 %token <string>  ID             /* testing343 */
 %token <string>  DOTTEDNAME     /* System.Object */
 %token <binstr>  QSTRING        /* "Hello World\n" */
 %token <string>  SQSTRING       /* 'Hello World\n' */
-%token <int32>   INT32          /* 3425 0x34FA  0352  */
-%token <int64>   INT64          /* 342534523534534      0x34FA434644554 */
+%token <int32>   INT32_V        /* 3425 0x34FA  0352  */
+%token <int64>   INT64_V        /* 342534523534534      0x34FA434644554 */
```

Comment:
They are conflicting with `INT32` and `INT64` typedef, so rename them.

Reply:
Yea, IIRC, the previous PR also had to do this.